src/Data/Fixed.purs

```diff
@@ -110,6 +111,12 @@ reifyPrecision n f = reifyPrecision (n - 1) (f <<< liftTenTimes) where
 -- | operations.
 newtype Fixed (precision :: Precision) = Fixed BigInt.BigInt
 
+instance decodeFixed :: Decode (Fixed precision) where
+  decode f = map (Fixed <<< BigInt.fromInt) (decode f)
```

Comment:
This isn't compatible with the `Encode` instance.

Reply:
Are you referring to Number vs Int?

Reply:
On the Haskell side, Aeson transforms `Integer`s to `Number`s.